projects/demo/src/modules/components/icon/examples/10/index.ts

```diff
@@ -0,0 +1,17 @@
+import {Component} from '@angular/core';
+import {changeDetection} from '@demo/emulate/change-detection';
+import {encapsulation} from '@demo/emulate/encapsulation';
+import {TuiIcon} from '@taiga-ui/core';
+
+@Component({
+    standalone: true,
+    imports: [TuiIcon],
+    templateUrl: './index.html',
```

Comment:
I think we can just inline single-line template 

```suggestion
    template: '<tui-icon [icon]="icon" />',
```

What do you think about it ?

Reply:
«Brevity is the sister of talent»